cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -217,9 +238,29 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
-  //TODO this is inelegant. However, as we're deprecating DocumentingTestBase, this will all be rewritten very soon
-  def assertIndexesOnLabels(label: String, expectedIndexes: List[List[String]], expectedIndexesAlt: List[List[String]]) {
-    assert(expectedIndexes === db.indexPropsForLabel(label) || expectedIndexesAlt === db.indexPropsForLabel(label))
+  @Test def use_index_with_bbox_query() {
+    executePreparationQueries(
+      (for(x <- -10 to 10; y <- -10 to 10) yield s"CREATE (:Person {location: point({x:$x, y:$y})})").toList ++ List(
+        "MATCH (n:Person {firstname: 'Andres'}) SET n.location = point({x: 1.2345, y: 5.4321})",
+        "MATCH (n:Person {firstname: 'Mark'}) SET n.location = point({y: 1.2345, x: 5.4321})"
+      )
+    )
+    profileQuery(
+      title = "Use index when executing a spatial bounding box search",
```

Comment:
Harmonize this title as per https://github.com/neo4j/neo4j-documentation/pull/241